client/components/document.vue

```diff
@@ -92,20 +98,36 @@
         required: true,
       },
     },
-
+    data() {
+      return {
+        publishDialog: false,
+      };
+    },
     computed: {
       document() {
         return Document.documents.findOne({
           _id: this.documentId,
         });
       },
+      currentUser() {
```

Comment:
I think we could extract this into a plugin. And then have `this.currentUser` returning `Meteor.user({username: 1, avatar: 1})`, and `this.currentUserId` returning `Meteor.userId()`. The latter is then useful when you just want to check if user is logged in.

Reply:
That's a good idea. Just to be clear, you are talking about a Vue plugin that we initialize doing something like `Vue.use(MeteorUserPlugin)`?

Reply:
Yea.